clar2wasm/src/lib.rs

```diff
@@ -15,8 +15,7 @@ mod words;
 
 #[cfg(feature = "developer-mode")]
 pub mod datastore;
-
-#[cfg(feature = "developer-mode")]
+#[cfg(any(test, feature = "developer-mode"))]
```

Comment:
What I mean is, you can use it from the tests without changing anything.

Leaving this as:
```
#[cfg(feature = "developer-mode")]
```
works fine.